helpers/shortcuts.ts

```diff
@@ -60,10 +60,16 @@ function transformKeys(keys: string[]) {
     if (Object.keys(keysMap).includes(key)) {
       return keysMap[key];
     }
-    return key.toUpperCase();
+    return toUpperCase(key);
   });
 }
 
+const toUpperCase = (key: string): string => {
```

Comment:
:bug: **Bug**
The `toUpperCase` function is incorrectly converting a character to uppercase by subtracting 40 from its char code. This approach is not only incorrect for converting lowercase letters to uppercase but also inefficient and will not work for all characters. For example, the char code for 'a' is 97, and subtracting 40 gives 57, which corresponds to '9', not 'A'. Instead, you should use the built-in `toUpperCase` method of the `String` object, which is optimized, handles all edge cases, and correctly converts a string to uppercase.